atlp-rwanda/e-comm-team-techsmith-fn
# #185528206 Integrate WebSockets for Automatic Stock Updates

## What does this PR do?
 This PR updates Websockets integrate for automatic stock

## Description of Task to be completed?

- Bug fix  Integrate WebSockets for Automatic Stock Updates

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout bg-integrate-webSockets-185528206`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?
- [#185528206](https://www.pivotaltracker.com/story/show/185528206)

## Screenshots (if appropriate)
